kura/org.eclipse.kura.net.admin/src/main/java/org/eclipse/kura/net/admin/visitor/linux/HostapdConfigWriter.java

```diff
@@ -90,16 +94,40 @@ public void visit(NetworkConfiguration config) throws KuraException {
                 .getModifiedNetInterfaceConfigs();
 
         for (NetInterfaceConfig<? extends NetInterfaceAddressConfig> netInterfaceConfig : netInterfaceConfigs) {
-            if (netInterfaceConfig.getType() == NetInterfaceType.WIFI) {
-                if (netInterfaceConfig.getName().startsWith("mon.")) {
-                    continue;
-                }
+
+            // skip monitor interface
```

Comment:
Please remove the comment.

Reply:
i'll keep it, to better explain what we are doing